projects/postgresql/build.sh

```diff
@@ -14,12 +14,24 @@
 # limitations under the License.
 #
 ################################################################################
-git apply ../add_fuzzers.diff
 cp -r $SRC/fuzzer src/backend/
-mkdir bld
+
+useradd fuzzuser
+chown -R fuzzuser .
 cd bld
 
-../configure --prefix /usr/local/
-make -j$(nproc)
+CC="" CXX="" CFLAGS="" CXXFLAGS="" su fuzzuser -c ../configure
```

Comment:
Why are you defining those variables? There should be empty by default.

Reply:
When running infra/helpyer.py build_fuzzers, those flags are set to specific values.

Reply:
They are set, but I'm surprised they aren't needed here. I guess CI passing https://github.com/google/oss-fuzz/pull/4357/checks?check_run_id=1026019910 means it works though.